src/message.rs

```diff
@@ -18,6 +20,18 @@ pub fn process(
     )
   })?;
 
+  if let Some(destination_path) = parameters.destination_path {
```

Comment:
If you manage that, remove `.with_message(&result),` if the destination path is present